runHarness/Parameters.pm

```diff
@@ -938,39 +938,40 @@ $parameters{"description"} = {
 	"showUsage" => 1,
 };
 
-$parameters{"runLength"} = {
-	"type"      => "=s",
-	"default"   => "medium",
-	"parent"    => "runProc",
-	"usageText" => "This is a shortcut for specifying the length of a run.\n\t"
-	  . "Allowable values:\n\t"
-	  . "short (120s, 180s, 60s)\n\t"
-	  . "medium (600s, 900s, 60s)\n\t"
-	  . "long ( 600s, 1800s, 120s)",
-	"showUsage" => 1,
-};
 $parameters{"rampUp"} = {
 	"type"    => "=i",
-	"default" => "",
+	"default" => "300",
 	"parent"  => "runProc",
 	"usageText" =>
-"This is the length of the ramp-up period for the run.\n\tIt overrides the ramp-up length set by the runLength parameter.\n\tThe number of users is slowly increased over the course of the ramp-up period.",
+"This is the length of the ramp-up period for the run.",
 	"showUsage" => 1,
 };
-$parameters{"steadyState"} = {
+$parameters{"numQosPeriods"} = {
```

Comment:
It is possible to specify 0 for numQosPeriods -- which although doesn't make sense, it kinda of runs but encounters uninitialized value issues.

Reply:
Good point.  I will add a test for this.